models/telegramResponse.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+type TelegramResponse struct {
+	Ok     bool `json:"ok"`
+	Result struct {
```

Comment:
It is better to define it outside the struct and use it within the struct to enhance readability.